packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -698,43 +700,59 @@ export default [
                 <Card>
                   <CardHeader>
                     {_(state.backupMode ? 'backup' : 'deltaBackup')}
+                    <Link
+                      className='pull-right'
+                      target='_blank'
+                      to='/settings/remotes'
+                    >
+                      <Button btnStyle='primary' size='small'>
```

Comment:
I don't think it's correct to use a `Button` inside a link, I think you better use the [`.btn` class from Bootstrap](https://v4-alpha.getbootstrap.com/components/buttons/#button-tags) directly on a `Link`.